src/components/AppletsTable/Table/Table.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react';
+import { Table as MuiTable, TableBody, TablePagination } from '@mui/material';
+
+import { DEFAULT_ROWS_PER_PAGE } from 'components/Table/Table.const';
+import { Head } from 'components/Table/Head';
+
+import { StyledTableContainer, StyledCellItem, StyledTableCellContent } from './Table.styles';
+import { Order, Row, TableProps } from './Table.types';
+import { FolderItem } from './FolderItem';
+import { AppletItem } from './AppletItem';
+
+export const Table = ({ columns, rows, orderBy: orderByProp, headerContent }: TableProps) => {
+  const [order, setOrder] = useState<Order>('asc');
+  const [orderBy, setOrderBy] = useState<string>(orderByProp);
+  const [page, setPage] = useState(0);
+
+  function descendingComparator(a: Row, b: Row, orderBy: string) {
```

Comment:
Could we create custom hook for descendingComparator, getComparator, handleRequestSort, handleChangePage functions or to move them to the Utils? Because they are the same as in Table component.